### PR TITLE
move loading plugins into Application

### PR DESCRIPTION
**Pull request by saeideng**
move loading plugins into Application
after releasing cake 3.6.6

### CODE REVIEW

File: config/bootstrap.php [hunk 1]
```diff
@@ -207,11 +207,3 @@
  * Plugin::load('Migrations'); //Loads a single plugin named Migrations
```
Comment (ADmad) [inline config/bootstrap.php]: Shouldn't this comment also be removed?

File: src/Application.php [hunk 2]
```diff
@@ -61,4 +61,33 @@ public function middleware($middlewareQueue)
...
+
+            $this->addPlugin('Migrations');
+        }
+
+        /*
+         * Only try to load DebugKit in development mode
+         * Debug Kit should not be installed on a production system
+         */
+        if (Configure::read('debug')) {
+            $this->addPlugin(\DebugKit\Plugin::class, ['bootstrap' => true]);
```
Comment (ADmad) [inline src/Application.php]: Specifying bootstrap => true shouldn't be necessary.

File: config/bootstrap.php (hunk 1 above)
Comment (saeideng) [reply config/bootstrap.php]: whole comments or just `Plugin::load('Migrations...` ?

**Comment by saeideng** [comment]
looks we should wait for releasing 3.6.7
I tested tests with https://github.com/cakephp/cakephp/pull/12300 
it was OK with ^^ patch

**Pull request merged**